encrypted-media/clearkey-mp4-playback-destroy-persistent-license.html

```diff
@@ -0,0 +1,53 @@
+<!doctype html>
+<html>
+  <head>
+    <meta charset=utf-8>
+    <meta name="timeout" content="long">
```

Comment:
A few tests don't have this.  It that intentional?

`<meta name="timeout" content="long"> `